src/nile/deployments.py

```diff
@@ -29,10 +29,8 @@ def register_class_hash(hash, network, alias):
     """Register a new deployment."""
     file = f"{network}.{DECLARATIONS_FILENAME}"
 
-    if class_exists(hash, network, alias):
-        raise Exception(
-            f"Duplicate hash {hash[:6]}...{hash[-6:]} or alias {alias} in {file}"
-        )
+    if hash_exists(hash, network):
```

Comment:
i liked the term _class_, although I see how the class could be represented by an alias. in any case "hash" is a bit generic in any crypto context 😅 

i vote for either `class_exists` or `class_hash_exists`, or similar